apps/web/pages/api/teams/googleworkspace/add.ts

```diff
@@ -0,0 +1,35 @@
+import { google } from "googleapis";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import getAppKeysFromSlug from "@calcom/app-store/_utils/getAppKeysFromSlug";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+
+const scopes = ["https://www.googleapis.com/auth/admin.directory.user.readonly"];
+
+let client_id = "";
+let client_secret = "";
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method === "GET") {
+    // Get appKeys from google-calendar
+    const appKeys = await getAppKeysFromSlug("google-calendar");
```

Comment:
It uses appKeys from google calendar. Now that's tricky. I think you can have google-calendar uninstalled?

Reply:
100% but the issue we have here is "google-calendar" yeah its an app. But the keys it stores are just google credentials no specificly for calendar. Hence them being using across all the google apis -> calendar,meet and now workspaces

Reply:
I will add a checkGlobalKeys as a proc. so we can check if its installed before we show the button